content/200-concepts/050-overview/300-prisma-in-your-stack/03-is-prisma-an-orm.mdx

```diff
@@ -222,50 +164,123 @@ The drawbacks of ORMs are not always apparent when you start using them. This se
 
 - With ORMs, you form an object graph representation of database tables; this is known as the [object-relational impedance mismatch](https://en.wikipedia.org/wiki/Object-relational_impedance_mismatch). Depending on your use-case, the problem you are solving may form a complex object graph which doesn't trivially map to a relational database. Synchronizing between two different representations of data, one in the relational database, and the other in-memory (with objects) is quite difficult. This is because objects are more flexible and varied in the way they can relate to each other compared to relational database records.
 - While ORMs handle the complexity associated with the problem, the synchronization problem doesn't go away. Any changes to the database schema or the data model require the changes to be mapped back to the other side. This burden is often on the developer. In the context of a team working on a project, database schema changes require coordination.
-- ORMs tend to have a large API surface due to the complexity they encapsulate. The flip side of not having to write SQL is that you spend a lot of time learning how to use the ORM. This applies to most abstractions, however without understanding the way the database works, understanding and improving slow queries can be difficult.
+- ORMs tend to have a large API surface due to the complexity they encapsulate. The flip side of not having to write SQL is that you spend a lot of time learning how to use the ORM. This applies to most abstractions, however without understanding how the database works, improving slow queries can be difficult.
 - Some _complex queries_ aren't supported by ORMs due to the flexibility that SQL offers. This problem is alleviated by raw SQL querying functionality in which you pass the ORM a SQL statement string and the query is run for you.
 
 Now that the costs and benefits of ORMs have been covered, you can better understand what Prisma is and how it fits in.
 
 ## Prisma
 
-Prisma is a **database toolkit** that makes working with databases easy for application developers. It currently consists of two main tools:
+Prisma is a database toolkit that makes working with databases easy for application developers. It consists of three tools:
```

Comment:
I wasn't sure of this because that would implicitly assume that Migrate is part of the ORM. But since it's next-generation, I think it's fair to include the three tools as part of the ORM.

Reply:
I wasn't sure of this because that would implicitly assume that Migrate is part of the ORM. 
Anyways, I'll update to next-generation ORM.